slackevents/inner_events.go

```diff
@@ -165,7 +165,7 @@ type LinkSharedEvent struct {
 	User             string        `json:"user"`
 	TimeStamp        string        `json:"ts"`
 	Channel          string        `json:"channel"`
-	MessageTimeStamp json.Number   `json:"message_ts"`
+	MessageTimeStamp string        `json:"message_ts"`
```

Comment:
Could you please add the reason for using `string` type(= `message_ts` sends more than just timestamps, contrary to the variable name)?

Reply:
absolutely